extensions/sql-database-projects/src/test/datasource.test.ts

```diff
@@ -14,6 +14,10 @@ describe('Data Sources: DataSource operations', function (): void {
 		await baselines.loadBaselines();
 	});
 
+	after(() => {
+		testUtils.deleteGeneratedTestFolder();
```

Comment:
is deleting the generated test folder after each test suite affecting another test running not a concern because the test suites run serially? I'm wondering if it's worth it for each test suite to make its own sub folder, then only delete its own generated test folder, or if it's unnecessary

Reply:
It would have been a problem if the tests would be running parallelly. Since it is serial, it cleans up after a set of tests are run, and goes on to create new folders for following tests.
I don't feel that it is necessary but I can surely create sub folder and clean after its own, the only problem comes with the base folder that might still remain undeleted at the end of the test run. For this, we would need to keep track of the last test that runs and that one eventually deleting it. But I feel that it is a little overhead to keep track, for which could be solved simply :D.

Reply:
as long as there aren't any race condition risks with creating and deleting the folder after each test suite, I'm ok with it. Any concerns with this, @Benjin?